### PR TITLE
Live Location Sharing - Reset zoom level while focusing a user (PSG-624)

### PR DESCRIPTION
**Pull request by onurays**
## Type of change

- [x] Feature
- [ ] Bugfix
- [ ] Technical
- [ ] Other :

## Content

When a user is selected on bottom sheet of maximized live location map view, we shouldn't preserve the current zoom level. Changing zoom level to a meaningful value will improve UX.

## Screenshots / GIFs

https://user-images.githubusercontent.com/1254401/179989271-83b390a8-2b7a-46c1-ad6e-9624855a5295.mp4

## Tests

- Share a live location
- Tap on the live location message in the timeline
- On maximized map screen tap on user to focus

## Tested devices

- [x] Physical
- [ ] Emulator
- OS version(s):

## Checklist

- [x] Changes has been tested on an Android device or Android emulator with API 21
- [x] UI change has been tested on both light and dark themes
- [x] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [x] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [x] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

### CODE REVIEW

File: vector/src/main/res/layout/fragment_location_live_map_view.xml [hunk 1]
```diff
@@ -8,8 +8,9 @@
 
     <View
         android:id="@+id/liveLocationPopupAnchor"
-        android:layout_width="40dp"
-        android:layout_height="40dp" />
+        android:layout_width="0dp"
```
Comment (Claire1817) [inline vector/src/main/res/layout/fragment_location_live_map_view.xml:11]: why did you change this ? 
this is not related with the bug ?

**Review by Claire1817 — COMMENTED** [review]
Simple question.

File: vector/src/main/res/layout/fragment_location_live_map_view.xml (hunk 1 above)
Comment (onurays) [reply vector/src/main/res/layout/fragment_location_live_map_view.xml:11]: We use the bottom point of this anchor for the "share location" popup. And previously we were placing this anchor aligned with the top of the marker. So previously we needed to shift the anchor to the bottom as the amount of the height of the marker.

Now, we always show the popup at the center of the screen, not related to the position of the marker. If we continue using 40dp height then it will give a gap of 40dp between the pin and the popup.

**Review by Claire1817 — APPROVED** [review]
